apps/remix-ide/src/app/tabs/locales/es/settings.json

```diff
@@ -36,5 +36,6 @@
   "settings.port": "PUERTO",
   "settings.projectID": "ID DEL PROYECTO",
   "settings.projectSecret": "SECRETO DE PROYECTO",
-  "settings.analyticsInRemix": "Analíticas en IDE Remix"
+  "settings.analyticsInRemix": "Analíticas en IDE Remix",
+  "settings.enableSaveEnvState": "Guardar el estado del entorno de implementación"
```

Comment:
i think this is fine if you just put the english translation next time.
Translations are made from the crowd-in interface.